desktop-app/src/store/index.ts

```diff
@@ -198,7 +219,7 @@ const schema = {
   },
   homepage: {
     type: 'string',
-    default: 'https://www.google.com/',
+    default: 'https://www.example.org/',
```

Comment:
Is this some debug change got into the commit?

Reply:
Yes, updated.